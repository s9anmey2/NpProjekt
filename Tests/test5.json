{
	"epsilon":0.0000000001,
	"width":5,
	"height":5,
	"allGuardsAdded":true,
	"column2row2initialValue":{
		"0":{
			"0":1.0
		}
	},
	"guards":[
		[0,3,4,2,4],
		[0,1,2,2,4],
		[0,1,4,0,1],
		[1,0,3,0,4],
		[2,3,4,1,2],
		[2,3,4,3,4],
		[2,0,2,1,4],
		[3,0,4,0,3]
	],
	"commands":[
		[0.0,0.0,0.0,0.000009],
		[0.0,0.0,0.0,0.00001],
		[0.0,0.0,0.0,0.00001],
		[0.0,0.0,0.0,0.00001],
		[0.0,0.0,0.0,0.4230],
		[0.0,0.0,0.0,0.4700],
		[0.0,0.0,0.0,0.4700],
		[0.0,0.0,0.0,0.4700]
	]
}
