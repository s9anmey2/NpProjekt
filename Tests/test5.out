0.03839723014737029 0.03839723014270114 0.038397130132077725 0.03526298885169629 0.03766145196035289
0.038397230147370356 0.03839723014482894 0.03839719681380688 0.03918096792967701 0.041846114434980944
0.03839723014737054 0.038397230147665794 0.038397246818845654 0.04353423353936268 0.046495802412617415
0.03839723014737068 0.038397230150148016 0.03839728015267816 0.04353412838180857 0.04649587423385671
0.03839723014737076 0.03839723015156638 0.03839729681873641 0.04353407580461534 0.04649591014357315
